library/lib_exec_node.ks

```diff
@@ -0,0 +1,43 @@
+@lazyglobal off.
+
+function MaxShipThrust{ 
+	// this function is needed as a workaround
+	// to kOS bug regarding ship thrust.
+	local engs is list().
+	list engines in engs.
+	local mth is 0.
+	for eng in engs{
+		if eng:VISP > 0 {
+			set mth to mth + eng:MAXTHRUST.
+		}
+	}
+	return mth.
+}
+
+function exec_node{
+	parameter nd.
+	
+	local mth is MaxShipThrust().
+	lock burntime to nd:deltav:mag/(mth/mass).
```

Comment:
Actually, I  think this PR should contain just the orbital library - the exec_node is not really finished and I included it here just because I noticed there was none in KSLib. But really, as it stands now, exec_node's quality is likely too far below quality to be included.